python/examples/05-send-msg/main.py

```diff
@@ -12,7 +12,7 @@ class Message(Model):
 @alice.on_interval(period=2.0)
 async def send_message(ctx: Context):
     msg = f"Hello there {bob.name} my name is {alice.name}."
-    await ctx.send(bob.address, Message(text=msg))
+    await ctx.send(bob.identifier, Message(text=msg))
```

Comment:
I kind of agree that this makes more sense in general, but for sending messages within a bureau, the identifier is not actually useful, so I would probably leave this as `bob.address`. It's tricky because we want to introduce the concept of identifier in a way that doesn't seem like we are changing something fundamental, which it really isn't. So I'd suggest using identifier only in the examples that actually use the Almanac and name service contracts.

Reply:
I agree